### ISSUE TITLE
Unity environment took too long to respond (Windows)

### ISSUE THREAD

**Issue by louisgv** [Unity-Technologies/obstacle-tower-challenge#13]
Output when running python run.py:

```
PS D:\Project\Github\otc-unity\obstacle-tower-challenge> python .\run.py
Traceback (most recent call last):
  File ".\run.py", line 26, in <module>
    env = ObstacleTowerEnv(args.environment_filename, docker_training=args.docker_training)
  File "C:\Users\lgvic\AppData\Local\Programs\Python\Python36\lib\site-packages\obstacle_tower_env.py", line 39, in __init__
    self._env = UnityEnvironment(environment_filename, worker_id, docker_training=docker_training)
  File "C:\Users\lgvic\AppData\Local\Programs\Python\Python36\lib\site-packages\mlagents_envs\environment.py", line 67, in __init__
    aca_params = self.send_academy_parameters(rl_init_parameters_in)
  File "C:\Users\lgvic\AppData\Local\Programs\Python\Python36\lib\site-packages\mlagents_envs\environment.py", line 493, in send_academy_parameters
    return self.communicator.initialize(inputs).rl_initialization_output
  File "C:\Users\lgvic\AppData\Local\Programs\Python\Python36\lib\site-packages\mlagents_envs\rpc_communicator.py", line 79, in initialize
    "The Unity environment took too long to respond. Make sure that :\n"
mlagents_envs.exception.UnityTimeOutException: The Unity environment took too long to respond. Make sure that :
         The environment does not need user interaction to launch
         The Academy and the External Brain(s) are attached to objects in the Scene
         The environment and the Python interface have compatible versions.
```

The Unity application actually open, but the level doesn't appear, it looks like below: 

![image](https://user-images.githubusercontent.com/6723574/52692013-8b570500-2f17-11e9-9e84-ea7502a95503.png)

I have a feeling that the Unity app was trying to connect with some port to reach the python app but for some reason cannot and crashed itself internally?

**Comment by tduval** [comment]
You should look at this issue : https://github.com/Unity-Technologies/obstacle-tower-env/issues/46
You will need to use the v1.0 of ObstacleTower data instead of the 1.1 for now.

**Comment by louisgv** [comment]
Thank you very much @tduval :)

**Comment by awjuliani** [comment]
Hi @louisgv 

We have also addressed this in a new version (v1.1.1) here: https://storage.googleapis.com/obstacle-tower-build/v1.1/obstacletower_v1.1.1_windows.zip which fixes this as well, and includes other fixes over v1.0
